App/features/signup.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Signup
+  In order to acces the application
+  As a visitor user
+  I need to be able to create an account
+
+  Background:
+    Given I am on /signup
+
+  Scenario Outline: Sign-up the application failed
+    And I fill in the input Username with "<username>"
+    And I fill in the input Email with "<email>"
+    And I fill in the input Password with "<password>"
+    And I fill in the input Confirm password with "<confirm_password>"
+    When I submit the form
+    Then I should be redirect to "/signup"
+
+  Examples:
+  | username | email                | password        | confirm_password |
+  | leo      | test.test@gmail.com  | Testpassword/10 | Testpassword/10  |
+  | leooooo  | test.test@gmail      | Testpassword/10 | Testpassword/10  |
+  | leooooo  | test.test@gmail.com  | password        | password         |
+  | leooooo  | test.test@gmail.com  | Testpassword/10 | password         |
```

Comment:
Thinking about that: in this scenarios you test each error input, it might be relevant to add a argument with the error message that should be displayed and ensure an error is displayed on the screen under the right input with the right text. Something like 
```
| username | email                | password        | confirm_password | error_input | error_message |
| leo      | test.test@gmail.com  | Testpassword/10 | Testpassword/10  | ...         | ...           |
```

Reply:
done